src/Plugins/OrientationAnalysis/src/OrientationAnalysis/utilities/OrientationUtilities.hpp

```diff
@@ -70,5 +73,37 @@ Eigen::Matrix<T, 3, 3, Eigen::RowMajor> EbsdLibMatrixToEigenMatrix(const EbsdLib
 
 std::string CrystalStructureEnumToString(uint32_t crystalStructureType);
 
+template <typename T>
+Eigen::Matrix<T, 3, 3, Eigen::RowMajor> GMatrixToEigenMatrix(T g[3][3])
```

Comment:
```suggestion
Eigen::Matrix<T, 3, 3, Eigen::RowMajor> GMatrixToEigenMatrix(const T g[3][3])
```
Since the argument here decays to a pointer, I think we should mark it `const`.